core/src/main/resources/hudson/model/Messages_fr.properties

```
@@ -91,45 +156,119 @@ Hudson.ReadPermission.Description=\
   Ce droit est utile quand vous ne voulez pas que les utilisateurs non authentifiés puissent voir les pages Jenkins \
   — retirez ce droit à l''utilisateur anonymous, puis \
   ajoutez le pseudo-utilisateur "authenticated" et accordez-lui le droit en lecture.
+Hudson.RunScriptsPermission.Description=\
+  Déprécié - veuillez utiliser le droit Overall/Administer à la place
```

```suggestion
  Déprécié - veuillez utiliser le droit "Overall/Administer" à la place
```
Quotes around english permission term for consistency.